### PR TITLE
Release 4.4.1

### PR DESCRIPTION
**Pull request by joaander**
## Description

Changes needed to release HOOMD-blue 4.4.1.

## Motivation and context

Make the latest fixes available to users.

## Checklist:

- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/CONTRIBUTING.rst).
- [x] I agree with the terms of the [**HOOMD-blue Contributor Agreement**](https://github.com/glotzerlab/hoomd-blue/blob/trunk-minor/ContributorAgreement.md).
- [x] My name is on the list of contributors (`sphinx-doc/credits.rst`) in the pull request source branch.

### CODE REVIEW

**Review by b-butler — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Minor and major releases:
   - See current actions usage with: `rg --no-filename --hidden uses: | awk '{$1=$1;print}' | sort | uniq`
   - Use global search and replace to update them to the latest tags
 - [ ] Check for new or duplicate contributors since the last release:
-  `comm -13 <(git log LAST_TAG --format="%aN <%aE>" | sort | uniq) <(git log --format="%aN <%aE>" | sort | uniq)`.
+  `comm -13 (git log v4.3.0 --format="%aN <%aE>" | sort | uniq | psub) (git log --format="%aN <%aE>" | sort | uniq | psub)`.
```
Comment (b-butler) [inline .github/ISSUE_TEMPLATE/release.md]: Just one question does bumpversion handle bumping to one version behind?

Comment (joaander) [reply .github/ISSUE_TEMPLATE/release.md]: I don't think so. But there is some shell magic one could do with `git tag` to get the most recent release.

**Comment by joaander** [comment]
I am delaying this release. There is one more performance bug to fix that increases HPMC performance another 50%.

**Review by tcmoore3 — APPROVED** [review]

**Pull request merged**